tests/unit/generators/globalConfig_generator/test_create_minimal_globalConfig.py

```diff
@@ -0,0 +1,122 @@
+from pytest import fixture
+from unittest.mock import patch, MagicMock
+from splunk_add_on_ucc_framework.generators.globalConfig_generator import (
+    MinimalGlobalConfig,
+)
+
+
+@fixture
+def global_config():
+    return None
+
+
+@fixture
+def app_manifest():
+    mock_manifest = MagicMock()
+    mock_manifest.get_addon_name.return_value = "test_addon"
+    mock_manifest.get_addon_version.return_value = "1.0.0"
+    mock_manifest.get_title.return_value = "Test title"
+    return mock_manifest
+
+
+@fixture
+def input_dir(tmp_path):
+    return str(tmp_path / "input_dir")
+
+
+@fixture
+def output_dir(tmp_path):
+    return str(tmp_path / "output_dir")
+
+
+@fixture
+def ucc_dir(tmp_path):
+    return str(tmp_path / "ucc_dir")
+
+
+@fixture
+def ta_name():
+    return "test_addon"
+
+
+@patch("addonfactory_splunk_conf_parser_lib.TABConfigParser")
```

Comment:
Why are we patching `TABConfigParser`?

Reply:
Since `_set_attributes` utilizes `TABConfigParser` to read `app.conf` and we do not have an actual `app.conf`, I have mocked it this.